x-pack/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -0,0 +1,707 @@
+      });
+
+      beforeEach(async () => {
+        testBed = await setup();
+
+        await act(async () => {
+          testBed.actions.selectTab('snapshots');
+          await nextTick(100);
+          testBed.component.update();
+        });
+      });
+
+      test('should display an empty prompt', () => {
+        const { exists } = testBed;
+
+        expect(exists('emptyPrompt')).toBe(true);
+      });
+
+      test('should invite the user to first register a repository', () => {
+        const { find, exists } = testBed;
+        expect(find('emptyPrompt.title').text()).toBe(
+          `You don't have any snapshots or repositories yet`
+        );
+        expect(exists('emptyPrompt.registerRepositoryButton')).toBe(true);
+      });
+    });
+
+    describe('when there are no snapshots but has some repository', () => {
+      beforeEach(async () => {
+        httpRequestsMockHelpers.setLoadSnapshotsResponse({
+          snapshots: [],
+          repositories: ['my-repo'],
+        });
+
+        testBed = await setup();
+
+        await act(async () => {
+          testBed.actions.selectTab('snapshots');
+          await nextTick(2000);
+          testBed.component.update();
+        });
+      });
+
+      test('should display an empty prompt', () => {
+        const { find, exists } = testBed;
+
+        expect(exists('emptyPrompt')).toBe(true);
+        expect(find('emptyPrompt.title').text()).toBe(`You don't have any snapshots yet`);
+      });
+
+      test('should have a link to the snapshot documentation', () => {
+        const { exists } = testBed;
+        expect(exists('emptyPrompt.documentationLink')).toBe(true);
+      });
+    });
+
+    describe('when there are snapshots and repositories', () => {
+      const snapshot1 = fixtures.getSnapshot({
+        repository: REPOSITORY_NAME,
+        snapshot: `a${getRandomString()}`,
+      });
+      const snapshot2 = fixtures.getSnapshot({
+        repository: REPOSITORY_NAME,
+        snapshot: `b${getRandomString()}`,
+      });
+      const snapshots = [snapshot1, snapshot2];
+
+      beforeEach(async () => {
+        httpRequestsMockHelpers.setLoadSnapshotsResponse({
+          snapshots,
+          repositories: [REPOSITORY_NAME],
+          errors: {},
+        });
+
+        testBed = await setup();
+
+        await act(async () => {
+          testBed.actions.selectTab('snapshots');
+          await nextTick(2000);
+          testBed.component.update();
+        });
+      });
+
+      test('should list them in the table', async () => {
+        const { table } = testBed;
+
+        const { tableCellsValues } = table.getMetaData('snapshotTable');
+        tableCellsValues.forEach((row, i) => {
+          const snapshot = snapshots[i];
+          expect(row).toEqual([
+            snapshot.snapshot, // Snapshot
+            REPOSITORY_NAME, // Repository
+            '23 May 2019 14:25:15', // Date created
```

Comment:
Not sure this is going to work consistently hard-coded like this, e.g., on my machine the output is `23 May 2019 02:25:15`

Reply:
Good catch! I wondered if it could create a problem and did not think of the hour format. I liked the idea of outputting the expected format but I guess I will have to add some js here. @silne30 @cjcenizal thoughts on this?